src/Kernel/Class.class.st

```diff
@@ -798,11 +798,11 @@ Class >> possibleVariablesFor: misspelled continuedFrom: oldResults [
 	results := misspelled correctAgainstDictionary: self classPool continuedFrom: oldResults.
 	self sharedPools do: [:pool |
 		results := misspelled correctAgainstDictionary: pool classPool continuedFrom: results ].
-	self superclass == nil
+	^ self superclass == nil
```

Comment:
i didn't replace == nil with isNil because im not sure it is a good idea. idk the consequences.

Reply:
I think it can safely be replaced by #ifNil:ifNotNil: here